packages/@ember/-internals/metal/lib/mixin.ts

```diff
@@ -421,7 +425,11 @@ export function applyMixin(obj: { [key: string]: any }, mixins: Mixin[], _hideKe
   let keys: string[] = [];
   let keysWithSuper: string[] = [];
 
-  (obj as any)._super = ROOT;
+  Object.defineProperty(obj, '_super', {
+    enumerable: false,
+    writable: true,
+    value: ROOT,
```

Comment:
Can you add `configurable: true` as well?

Reply:
good catch!